addons_engine/addons-monitor/addons_monitor.py

```diff
@@ -0,0 +1,40 @@
+import logging
+import os
+
+import socketio
+from services import monitor_service
+
+ADDONS_MANAGER_ADDR = os.environ.get("ADDONS_MANAGER_ADDR") or "http://localhost:11101"
+
+# Syncronous socketio client
+socketio = socketio.Client()
+
+if __name__ == "__main__":
+    addon_manager = None
+
+    def on_receive_manager_id(manager_id):
+        global addon_manager
+
+        logging.info(f"Received manager id: {manager_id}")
+        addon_manager = monitor_service.init_addons_monitor(manager_id, socketio)
+
+    def on_connect():
+        logging.info("Successfully connected to addon manager!")
+        socketio.emit("get_manager_id")
+
+    socketio.on("connect", on_connect)
+    socketio.on("receive_manager_id", on_receive_manager_id)
+
+    # will keep trying to connect for 30 seconds
+    socketio.connect(ADDONS_MANAGER_ADDR, wait_timeout=30)
+
+    # wait until addon_manager is initialized
+    while not addon_manager:
```

Comment:
Do you think it makes sense adding a sleep cmd here instead of just a pass? maybe this will be less "demanding" on the system?

Reply:
yeah I guess, it shouldn't loop forever I agree. I'll adjust.